src/redux/reducers/tweets.ts

```diff
@@ -53,6 +70,9 @@ const tweetsSlice = createSlice<TweetsStore, SliceCaseReducers<TweetsStore>>({
       }
       store.feedTweets.isLoading = false;
     });
+    builder.addCase(createTweet.fulfilled, (store) => {
+      store.feedTweets.isLoading = false;
+    });
```

Comment:
1. please keep order by action not by state, like:
- createTweet.pending
- createTweet.fulfilled
- createTweet.rejected
- fetchFeedTweets.pending
- fetchFeedTweets.fulfilled
- fetchFeedTweets.rejected

2. I don't see handler for rejected state of "createTweet"

Reply:
1. I'm sorry, but I do not understand clearly.
2. We decided that it is unecessary, because we catch err at a higher level (as I understood from the conversation)

Reply:
conversation with who?
you should handle it in any case, because if you not handle this case, and your request will fail, isLoading state will keep true, and never change

Reply:
order of your code is not clear, please do like this
```
    builder.addCase(fetchFeedTweets.pending, (store) => {
      store.feedTweets.isLoading = true;
    });
    builder.addCase(fetchFeedTweets.fulfilled, (store, { payload }) => {
      if (store.feedTweets.docs.length === 0 || payload.page !== 1) {
        // I don't know why, but first page we have twice
        // and only such way I resolved this problem
        store.feedTweets = {
          ...store.feedTweets,
          ...payload,
          docs: [...store.feedTweets.docs, ...payload.docs],
        };
      }
      store.feedTweets.isLoading = false;
    });
    builder.addCase(fetchFeedTweets.rejected, (store) => {
      store.feedTweets.isLoading = false;
      store.feedTweets.error = "Failed to fetch tweets for feed";
    });
    builder.addCase(createTweet.pending, (store) => {
      store.feedTweets.isLoading = true;
    });
    builder.addCase(createTweet.fulfilled, (store) => {
      store.feedTweets.isLoading = false;
    });
```

Reply:
Thanks, i got it